# start
geometry_msgs/PoseStamped start

int8 planning_mode
float64 allocated_planning_time

# added for full body experiments without gazebo
float64[] rarm_start
float64[] larm_start
float64[] body_start
float64[] rarm_goal
float64[] larm_goal
float64[] body_goal

# goal
geometry_msgs/PoseStamped goal

# tolerance
float64 xyz_tolerance
float64 roll_tolerance
float64 pitch_tolerance
float64 yaw_tolerance

# right arm position on the object (in the object frame)
geometry_msgs/PoseStamped rarm_object

# left arm position on the object (in the object frame)
geometry_msgs/PoseStamped larm_object

float64 initial_eps
float64 final_eps
float64 dec_eps

---

# two arm trajectory
trajectory_msgs/JointTrajectory trajectory
trajectory_msgs/JointTrajectory body_trajectory

# planning stats
string[] stats_field_names
float64[] stats
